packages/legacy/README.md

```diff
@@ -0,0 +1,94 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-legacy
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-legacy
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-legacy
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-legacy
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-legacy
+
+Add `export` declarations to legacy non-module scripts.
+
+## Installation
+
+```bash
+npm install --save-dev rollup-plugin-legacy
+```
+
+## Motivation
```

Comment:
This is a good section and good info that we don't want to remove - generally speaking we want to keep extra sections like this. But let's move this down below `Options` once that section is added.